components/portal/src/components/image/ImageDeleteDialog.js

```diff
@@ -44,17 +45,22 @@ class ImageDeleteDialog extends React.Component {
 
     handleDeleteImage = () => {
         const self = this;
-        const {globalState, image, org} = self.props;
+        const {globalState, image, org, history} = self.props;
         NotificationUtils.showLoadingOverlay(`Deleting image ${org}/${image}`, globalState);
         HttpUtils.callHubAPI(
             {
-                url: `/images/${org}/${image}}`,
+                url: `/images/${org}/${image}`,
                 method: "DELETE"
             },
             globalState
         ).then(() => {
             self.handleClose();
             NotificationUtils.hideLoadingOverlay(globalState);
+            if (history.length > 2) {
```

Comment:
Can you please verify whether this is correct. Seems like it might be wrong IINM.